mods/cnc/chrome/dialogs.yaml

```diff
@@ -156,19 +156,26 @@ Container@CONFIRM_PROMPT:
 					Height: 25
 					Font: Bold
 					Align: Center
-		Button@CANCEL_BUTTON:
-			Key: escape
+		Button@CONFIRM_BUTTON:
+			Key: return
+			X: 360
 			Y: 89
 			Width: 140
 			Height: 35
-			Text: Cancel
-		Button@CONFIRM_BUTTON:
-			Key: return
-			X: 230
+			Text: Abort
+		Button@OTHER_BUTTON:
+			Key: r
+			X: 180
```

Comment:
The UI standard for TD is to group related buttons with 10px clear space between them, with 'forward' moving actions to the right, and 'back' moving actions to the left.  Please move this 30px to the right to fit in with that.